x-pack/plugins/ml/server/models/job_service/new_job_caps/field_service.ts

```diff
@@ -103,7 +104,7 @@ class FieldsService {
   }
 
   private isCounterField(field: estypes.FieldCapsFieldCapability) {
```

Comment:
If I understood this correctly, this is a different type coming from the field caps service, not DataView. The types estypes.FieldCapsFieldCapability are different, and the check is also different (e.g. `time_series_metric` vs `timeSeriesMetric`). We can potentially add check for both in one function `isCounterTimeSeriesMetric`? Since this check is only used one time in 3 packages I didn't think it was necessary to have its own util in the package.

Reply:
I missed the differences, I'm fine to keep as is!